scripts/components/ganttChartDashboards/build.sh

```diff
@@ -10,8 +10,12 @@
 set -ex
 
 # vars / libs
-. ../../../lib/shell/file_management.sh
-PLUGIN_PATH=$PWD
+SCRIPT_DIR=`dirname $(realpath $0)`
+. $SCRIPT_DIR/../../../lib/shell/file_management.sh
+# For hybrid plugin it actually resides in 'dashboards-visualizations/gantt-chart'
```

Comment:
@peterzhuamazon does this work for windows?

Reply:
I think so, it is using the bash.exe anyway during the run.
Tho I do not have a server to test for now.
If needed, I can do a verification tomorrow once I obtain a server. Thanks.

Reply:
After talking to @prudhvigodithi we will use the jenkins pipeline to test it.